File: code_challenge_solution/javascript/132 Name generator.js
```javascript
// Array of possible first names
const firstNames = ['John', 'Jane', 'David', 'Emily', 'Michael', 'Sarah', 'Christopher', 'Jessica', 'Matthew', 'Olivia'];

// Array of possible last names
const lastNames = ['Smith', 'Johnson', 'Brown', 'Jones', 'Miller', 'Davis', 'Garcia', 'Rodriguez', 'Wilson', 'Martinez'];

// Function to generate a random name
function generateRandomName() {
  const randomFirstName = firstNames[Math.floor(Math.random() * firstNames.length)];
  const randomLastName = lastNames[Math.floor(Math.random() * lastNames.length)];

  return randomFirstName + ' ' + randomLastName;
}

// Generate and display a random name
console.log(generateRandomName());
```

File: code_challenge_solution/java/132 Name generator.java
```java
import java.util.Random;

public class NameGenerator {
    private static final String[] FIRST_NAMES = {"John", "Michael", "Emily", "Emma", "Olivia", "Sophia", "Liam", "Jacob"};
    private static final String[] LAST_NAMES = {"Smith", "Johnson", "Williams", "Brown", "Jones", "Miller", "Davis", "Garcia"};

    public static void main(String[] args) {
        String randomFirstName = getRandomName(FIRST_NAMES);
        String randomLastName = getRandomName(LAST_NAMES);

        System.out.println("Random Name: " + randomFirstName + " " + randomLastName);
    }

    private static String getRandomName(String[] names) {
        Random random = new Random();
        int randomIndex = random.nextInt(names.length);
        return names[randomIndex];
    }
}
```